EdFi.Buzz.UI.Angular/src/app/Services/authentication.service.ts

```diff
@@ -42,6 +43,14 @@ export class AuthenticationService {
     // remove user from local storage to log user out
     this.storage.removeItem('currentUser');
     this.currentUserSubject.next(null);
+    //clear graphql cache
+    this.clearApolloStorage();
+  }
+
+  private clearApolloStorage() {
```

Comment:
I think this method should be in a new apolloClientHelper class or something like that so we can reuse it in other services.

Reply:
Done